nx-X11/include/extensions/Imakefile

```diff
@@ -47,9 +47,6 @@ RENDERHEADERS = render.h renderproto.h
 #if BuildXfixes || BuildXfixesLibrary
 XFIXESHEADERS = xfixeswire.h xfixesproto.h
 #endif
-#if BuildDamage || BuildDamageLibrary
-DAMAGEHEADERS = damagewire.h damageproto.h
```

Comment:
On  Di 12 Mai 2015 05:45:34 CEST, Mihai Moldovan wrote:

>> @@ -47,9 +47,6 @@ RENDERHEADERS = render.h renderproto.h
>>  #if BuildXfixes || BuildXfixesLibrary
>>  XFIXESHEADERS = xfixeswire.h xfixesproto.h
>>  #endif
>> -#if BuildDamage || BuildDamageLibrary
>> -DAMAGEHEADERS = damagewire.h damageproto.h
>
> Both files can probably be removed from `debian/copyright.in`  
> (although I remember we've been talking about that... but not the  
> outcome of this conversation. Leaving it in copyright.in is probably  
> fine anyway.)
>

debian/copyright.in is script-created. We will re-run this script  
prior to a release.
-- 

DAS-NETZWERKTEAM
mike gabriel, herweg 7, 24357 fleckeby
fon: +49 (1520) 1976 148

GnuPG Key ID 0x25771B31
mail: mike.gabriel@das-netzwerkteam.de, http://das-netzwerkteam.de

freeBusy:
https://mail.das-netzwerkteam.de/freebusy/m.gabriel%40das-netzwerkteam.de.xfb